conf/sage.config

```diff
@@ -1,19 +1,52 @@
+// Config profile metadata
 params {
   config_profile_description = 'The Sage Bionetworks profile'
   config_profile_contact = 'Bruno Grande (@BrunoGrandePhD)'
   config_profile_url = 'https://github.com/Sage-Bionetworks-Workflows'
 }
 
+// Leverage us-east-1 mirror of select human and mouse genomes
+params {
+  igenomes_base = 's3://sage-igenomes/igenomes'
+  max_memory    = '128.GB'
+  max_cpus      = 16
+  max_time      = '240.h'
+}
+
+// Enable retries globally for certain exit codes
+process {
+  errorStrategy = { task.exitStatus in [143,137,104,134,139,247] ? 'retry' : 'finish' }
```

Comment:
I wonder if there's a way to use this same strategy to adjust whether to increase the time allocation or memory based on the exit code. For example, on AWS Batch, exit code 143 means that the job reached its time limit.

Reply:
That's a very good question actuallly... might be worth asking on slack, it wouldn't surprise me if someone hasn't already had that issue.